mphys/builder.py

```diff
@@ -112,3 +112,18 @@ def get_ndof(self):
             number of degrees of freedom of each node in the domain
         """
         return -1
+
+    def get_tagged_indices(self, tags):
+        """
+        Method that returns grid IDs for a list of body/boundary tags.
+
+        Parameters
+        ----------
+        tags : list of integers or strings
+
+        Returns
+        -------
+        grid_ids : list
+            list of grid IDs that correspond to given body/boundary tags
```

Comment:
@Asthelen, could we add a note to the docstring what returning `None` by default implies?

Does this mean that all nodes will be used or none of them?

If its the latter does this represent a breaking change for older builders?